tests/test_basic.py

```diff
@@ -273,3 +273,10 @@ def test_py7zr_list_values():
     assert file_list[1].crc32 == 0xB36AAEDB
     assert file_list[2].crc32 == 0xDCBF8D07
     assert file_list[3].crc32 == 0x80FC72BE
+
+
+@pytest.mark.basic
+def test_multiple_uses():
+    with py7zr.SevenZipFile(os.path.join(testdata_path, "test_multiple.7z"), "w") as archive:
+        archive.writestr('1', '1.txt')
```

Comment:
When apply `black` command, it changed
```suggestion
        archive.writestr("1", "1.txt")
```